src/storagesync/azext_storagesync/custom.py

```diff
@@ -0,0 +1,201 @@
+    body = {}
+    body['storage_account_resource_id'] = storage_account_resource_id  # str
+    body['azure_file_share_name'] = azure_file_share_name  # str
+    body['storage_account_tenant_id'] = storage_account_tenant_id  # str
+    return sdk_no_wait(no_wait, client.create, resource_group_name=resource_group_name, storage_sync_service_name=storage_sync_service_name, sync_group_name=sync_group_name, cloud_endpoint_name=cloud_endpoint_name, parameters=body)
+
+
+def delete_storagesync_cloud_endpoint(client,
+                                      resource_group_name,
+                                      storage_sync_service_name,
+                                      sync_group_name,
+                                      cloud_endpoint_name,
+                                      no_wait=False):
+    return sdk_no_wait(no_wait, client.delete, resource_group_name=resource_group_name, storage_sync_service_name=storage_sync_service_name, sync_group_name=sync_group_name, cloud_endpoint_name=cloud_endpoint_name)
+
+
+def get_storagesync_cloud_endpoint(client,
+                                   resource_group_name,
+                                   storage_sync_service_name,
+                                   sync_group_name,
+                                   cloud_endpoint_name):
+    return client.get(resource_group_name=resource_group_name, storage_sync_service_name=storage_sync_service_name, sync_group_name=sync_group_name, cloud_endpoint_name=cloud_endpoint_name)
+
+
+def list_storagesync_cloud_endpoint(client,
+                                    resource_group_name,
+                                    storage_sync_service_name,
+                                    sync_group_name):
+    return client.list_by_sync_group(resource_group_name=resource_group_name, storage_sync_service_name=storage_sync_service_name, sync_group_name=sync_group_name)
+
+
+def create_storagesync_server_endpoint(client,
+                                       resource_group_name,
+                                       storage_sync_service_name,
+                                       sync_group_name,
+                                       server_endpoint_name,
+                                       server_resource_id,
+                                       server_local_path,
+                                       cloud_tiering=None,
+                                       volume_free_space_percent=None,
+                                       tier_files_older_than_days=None,
+                                       offline_data_transfer=None,
+                                       offline_data_transfer_share_name=None,
+                                       no_wait=False):
+    body = {}
+    body['server_resource_id'] = server_resource_id  # str
+    body['server_local_path'] = server_local_path  # str
+    body['cloud_tiering'] = cloud_tiering  # str
+    body['volume_free_space_percent'] = volume_free_space_percent  # number
```

Comment:
what's the meaning of number? should it be `type=int`?

Reply:
should be int,, number is generated from codegen,

---
In reply to: [392870777](https://github.com/Azure/azure-cli-extensions/pull/1389#discussion_r392870777) [](ancestors = 392870777)